packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -223,13 +224,19 @@ describe(`E2E: Web frameworks`, () => {
 		},
 	};
 
-	test.concurrent.each(Object.keys(frameworkTests))(
```

Comment:
When this line is removed will the tests still run concurrently?

Reply:
I suppose we could also just early exit from the function instead of using `skipIf` should we want to keep it

Reply:
> When this line is removed will the tests still run concurrently?
No sorry.... Igor did pick this up as well, sorry it was an oversight on my part 😓

Reply:
> I suppose we could also just early exit from the function instead of using skipIf should we want to keep it

Yes! I did want to do that in the first place! but I wouldn't want to just exit (and have the test pass) I would like to exit and mark the test as skipped (cypress has this sort of functionality for example) but unfortunately it doesn't look like Vitest offers such functionality 😢

Reply:
Fixed by adding a `.concurrent` in the `describe` as that has the same effect 🙂  ([docs](https://vitest.dev/guide/features.html#running-tests-concurrently))